.github/workflows/Dockerfile.dev

```diff
@@ -31,4 +32,12 @@ RUN wget -c https://musl.cc/x86_64-linux-musl-cross.tgz && \
     rm ./x86_64-linux-musl-cross.tgz && \
     rm ./aarch64-linux-musl-cross.tgz
 
+### armhf deps
+RUN sudo dpkg --add-architecture armhf && \
```

Comment:
Remove all the `sudo`, we're `root` and `sudo` is not installed.

Reply:
im updating the dockerfile dev to include rust musl target. ☕